packages/commands/src/solana/deposit.ts

```
@@ -0,0 +1,175 @@
+import GATEWAY_PROD_IDL from "@zetachain/protocol-contracts-solana/prod/idl/gateway.json";
+import { ethers } from "ethers";
+import { z } from "zod";
+
+import { SOLANA_TOKEN_PROGRAM } from "../../../../types/shared.constants";
+import { handleError, validateAndParseSchema } from "../../../../utils";
+import {
+  keypairFromMnemonic,
+  keypairFromPrivateKey,
+  solanaDepositOptionsSchema,
+  createSolanaCommandWithCommonOptions,
+} from "../../../../utils/solana.commands.helpers";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)